cylc/uiserver/data_store_mgr.py

```diff
@@ -90,6 +90,14 @@ async def sync_workflow(self, w_id, *args, **kwargs):
         )
         await self.entire_workflow_update(ids=[w_id])
 
+    async def register_workflow(self, w_id, name, owner):
+        flow = deepcopy(DATA_TEMPLATE)[WORKFLOW]
+        flow.id = w_id
+        flow.name = name
+        flow.owner = owner
+        flow.status = 'stopped'
+        self.data[w_id] = flow
```

Comment:
This breaks the UIS launch, needs to change to:
```
sutherlander@cortex-vbox:cylc-uiserver$ git diff
diff --git a/cylc/uiserver/data_store_mgr.py b/cylc/uiserver/data_store_mgr.py
index dcd5aed..6f43a7f 100644
--- a/cylc/uiserver/data_store_mgr.py
+++ b/cylc/uiserver/data_store_mgr.py
@@ -91,12 +91,13 @@ class DataStoreMgr:
         await self.entire_workflow_update(ids=[w_id])
 
     async def register_workflow(self, w_id, name, owner):
-        flow = deepcopy(DATA_TEMPLATE)[WORKFLOW]
+        data = deepcopy(DATA_TEMPLATE)
+        flow = data[WORKFLOW]
         flow.id = w_id
         flow.name = name
         flow.owner = owner
         flow.status = 'stopped'
-        self.data[w_id] = flow
+        self.data[w_id] = data
```
Otherwise the workflow element `PbWorkflow` is set as the data.